java/ql/src/semmle/code/java/security/XPath.qll

```diff
@@ -0,0 +1,64 @@
+/** Provides classes to reason about XPath vulnerabilities. */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
```

Comment:
The only dataflow/tainttracking reference in this file is `DataFlow::Node`, so we can change this to
```suggestion
import semmle.code.java.dataflow.DataFlow
```

Reply:
Good catch, thanks! Fixed in ea74205.